_scss/_uswds-theme-custom-styles.scss

```
@@ -21,3 +21,19 @@ i.e.
 */
 
 @import "components/glossary";
+
+.usa-footer-primary-section {
+  padding: 1rem 0;
+}
+
+.usa-footer--text {
+  @include u-font("sans", "xs");
+}
+
+.usa-footer-secondary-section {
+  background-color: #000;
```

The way USWDS v2 works is definitely new to me, and I'm still learning best practices, but if you don't want to hard code the hex value here I believe you could do something like this instead:
```suggestion
  @include u-bg("base-darkest");
```

---

cool, added this suggested in this commit: c929074